src/vaadin-chart.html

```diff
@@ -1435,83 +1429,81 @@
           return (this.type === 'bar' ? oppositeBar : opposite).indexOf(this.categoryPosition) >= 0;
         }
 
-        __updateCategoryPosition() {
-          if (!this.configuration) {
+        __updateCategoryPosition(categoryPosition, config) {
+          if (categoryPosition === undefined || !config) {
             return;
           }
 
           const validPositions = ['left', 'right', 'top', 'bottom'];
 
-          if (validPositions.indexOf(this.categoryPosition) < 0) {
+          if (validPositions.indexOf(categoryPosition) < 0) {
             console.warn(`<vaadin-chart> Acceptable "category-position" values are ${validPositions}`);
```

Comment:
Yes, we have to check for `undefined` in multi-property observers as otherwise they will run on initialization (unlike single property observers which wait for property to be set).

I think we can consider setting `null` a correct approach to reset a property.